pkg/controller/clusterdeployment/clusterdeployment_controller.go

```diff
@@ -180,6 +180,7 @@ type ReconcileClusterDeployment struct {
 // +kubebuilder:rbac:groups=hive.openshift.io,resources=clusterimagesets,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=hive.openshift.io,resources=clusterimagesets/status,verbs=get;update;patch
 func (r *ReconcileClusterDeployment) Reconcile(request reconcile.Request) (reconcile.Result, error) {
+
```

Comment:
I'm fine with this, but this feels like it was accidental. Do you really want this?